rosidl_generator_c/include/rosidl_generator_c/message_type_support_struct.h

```diff
@@ -28,19 +28,8 @@ typedef struct ROSIDL_PUBLIC_TYPE rosidl_message_type_support_t
   const void * data;
 } rosidl_message_type_support_t;
 
-/* This macro is used to create the symbol of the get_message_type_support
- * function for a specific message type. The library of the message package
- * which defines a given message will provide the symbol to which this macro
- * expands.
- */
-#define ROSIDL_GET_TYPE_SUPPORT(MsgPkgName, MsgSubfolder, MsgName) \
-  rosidl_get_message_type_support__ ## MsgPkgName ## __ ## MsgSubfolder ## __ ## MsgName()
-
-#define ROSIDL_GET_MESSAGE_TYPE_SUPPORT(MsgPkgName, MsgName) \
-  ROSIDL_GET_TYPE_SUPPORT(MsgPkgName, msg, MsgName)
-
```

Comment:
Also, because of how these macros need to work, we cannot declare them here and put the implementation specific version of them elsewhere (as we did with template functions in C++).